apps/web/components/ui/Review/Review.vue

```diff
@@ -3,8 +3,10 @@
     <div class="w-full flex">
       <div class="w-2/3 text-xs truncate text-neutral-400 mb-2">
         <span class="mr-2 text-xs text-neutral-700" data-testid="review-item-authorName">{{ reviewAuthor }}</span>
-        <SfIconCheck v-if="verifiedPurchase" size="xs" class="mr-1" />
-        {{ t('review.verifiedPurchase') }}
+        <span class="text-green-800">
+          <SfIconCheck v-if="verifiedPurchase" size="xs" class="mr-1" />
+          {{ t('review.verifiedPurchase') }}
+        </span>
```

Comment:
```suggestion
        <span v-if="verifiedPurchase" class="text-green-800">
          <SfIconCheck size="xs" class="mr-1" />
          {{ t('review.verifiedPurchase') }}
        </span>
```

Reply:
If the condition is false then it would have created an empty element

Reply:
done